html5/shared/utils.js

```diff
@@ -76,6 +76,11 @@ function hasOwn (obj, key) {
  */
 
 function bind (fn, ctx) {
+  if (typeof fn !== 'function') {
```

Comment:
OK. There is already a type check in directive `bindEvents`, I just change the `debug` info to an `error`.